### PR TITLE
Enable saml signature verification by default

### LINKED ISSUE #1226: Enable SAML signature by default
**What would you like to be added**:
Enable SAML signature by default

**Why is this needed**:

### PR DESCRIPTION
**Pull request by msenmurugan**
resolves #1226

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1227#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2020.2.x/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1227#issuecomment-586046981):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by yashbhutwala** [comment]
requires changes.

**Comment by yashbhutwala** [comment]
This needs to be there for 12.x, but not for 2020.2.x (since it will be enabled by default in the container itself)

File: pkg/apps/blackduck/latest/containers/environs.go [hunk 1]
```diff
@@ -37,4 +43,8 @@ func GetHubKnobs() map[string]string {
 		"DATA_RETENTION_IN_DAYS":            "180",
 		"MAX_TOTAL_SOURCE_SIZE_MB":          "4000",
 	}
+	if versionSAMLSignatureVersionSupportedByDefault, _ := util.IsVersionGreaterThanOrEqualTo(version, 2020, time.February, 0); versionSAMLSignatureVersionSupportedByDefault {
```
Comment (yashbhutwala) [inline pkg/apps/blackduck/latest/containers/environs.go]: Only set it if it's exactly 2019.12.1.  For future versions, this is set by default in containers.

**Pull request merged**